diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index 95c708caa..1f5cd83f0 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v2.2.1.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v2.2.1/SVT-AV1-v2.2.1.tar.gz
-SVT-AV1.FETCH.sha256  = d02b54685542de0236bce4be1b50912aba68aff997c43b350d84a518df0cf4e5
+SVT-AV1.FETCH.url     = https://github.com/Nj0be/HandBrake-SVT-AV1-SPY/releases/download/contribs/SVT-AV1-PSY-v2.2.0.tar.gz
+SVT-AV1.FETCH.sha256  = 622b1490cb63d3faad1fa2bc84160bf172b89396934792e1d743f6d4a891b619
 
 SVT-AV1.GCC.args.c_std =
