diff --git a/libhb/encsvtav1.c b/libhb/encsvtav1.c
index 2f16984d5..f7913036d 100644
--- a/libhb/encsvtav1.c
+++ b/libhb/encsvtav1.c
@@ -188,6 +188,14 @@ int encsvtInit(hb_work_object_t *w, hb_job_t *job)
     {
         param->tune = 2;
     }
+    else if (job->encoder_tune != NULL && strstr("subjective ssim", job->encoder_tune) != NULL)
+    {
+        param->tune = 3;
+    }
+    else if (job->encoder_tune != NULL && strstr("still picture", job->encoder_tune) != NULL)
+    {
+        param->tune = 4;
+    }
     else if (job->encoder_tune != NULL && strstr("psnr", job->encoder_tune) != NULL)
     {
         param->tune = 1;

diff --git a/libhb/handbrake/av1_common.h b/libhb/handbrake/av1_common.h
index 6d53a9dac..31158f6d0 100644
--- a/libhb/handbrake/av1_common.h
+++ b/libhb/handbrake/av1_common.h
@@ -31,7 +31,7 @@ static const char * const hb_av1_svt_preset_names[] =
 
 static const char * const hb_av1_svt_tune_names[] =
 {
-    "vq", "psnr", "ssim", "fastdecode", NULL
+    "vq", "psnr", "ssim", "subjective ssim", "still picture", "fastdecode", NULL
 };
 
 static const char * const hb_av1_svt_profile_names[] =
