diff --git a/libhb/encsvtav1.c b/libhb/encsvtav1.c
index a2f18034f..e525efb07 100644
--- a/libhb/encsvtav1.c
+++ b/libhb/encsvtav1.c
@@ -184,10 +184,19 @@ int encsvtInit(hb_work_object_t *w, hb_job_t *job)
         }
     }
 
+
     if (job->encoder_tune != NULL && strstr("ssim", job->encoder_tune) != NULL)
     {
         param->tune = 2;
     }
+    else if (job->encoder_tune != NULL && strstr("subjective ssim", job->encoder_tune) != NULL)
+    {
+        param->tune = 3;
+    }
+    else if (job->encoder_tune != NULL && strstr("still picture", job->encoder_tune) != NULL)
+    {
+        param->tune = 4;
+    }
     else if (job->encoder_tune != NULL && strstr("psnr", job->encoder_tune) != NULL)
     {
         param->tune = 1;

diff --git a/libhb/handbrake/av1_common.h b/libhb/handbrake/av1_common.h
index 79d19c1f0..713ab04b0 100644
--- a/libhb/handbrake/av1_common.h
+++ b/libhb/handbrake/av1_common.h
@@ -31,7 +31,7 @@ static const char * const av1_svt_preset_names[] =
 
 static const char * const av1_svt_tune_names[] =
 {
-    "psnr", "ssim", "fastdecode", NULL
+    "psnr", "ssim", "subjective ssim", "still picture", "fastdecode", NULL
 };
 
 static const char * const av1_svt_profile_names[] =
