diff --git a/libhb/handbrake/av1_common.h b/libhb/handbrake/av1_common.h
index 6d53a9dac..0d6abb3fd 100644
--- a/libhb/handbrake/av1_common.h
+++ b/libhb/handbrake/av1_common.h
@@ -26,7 +26,7 @@ static const int          hb_av1_level_values[] = {
 
 static const char * const hb_av1_svt_preset_names[] =
 {
-    "10", "9", "8", "7", "6", "5", "4", "3", "2", "1", "0", "-1", NULL
+    "13", "12", "11", "10", "9", "8", "7", "6", "5", "4", "3", "2", "1", "0", "-1", "-2", "-3", NULL
 };
 
 static const char * const hb_av1_svt_tune_names[] =
