diff --git a/contrib/svt-av1/module.defs b/contrib/svt-av1/module.defs
index 712ec75b7..390cf55d4 100644
--- a/contrib/svt-av1/module.defs
+++ b/contrib/svt-av1/module.defs
@@ -1,9 +1,8 @@
 $(eval $(call import.MODULE.defs,SVT-AV1,svt-av1))
 $(eval $(call import.CONTRIB.defs,SVT-AV1))
 
-SVT-AV1.FETCH.url     = https://github.com/HandBrake/HandBrake-contribs/releases/download/contribs2/SVT-AV1-v3.0.1.tar.gz
-SVT-AV1.FETCH.url    += https://gitlab.com/AOMediaCodec/SVT-AV1/-/archive/v3.0.1/SVT-AV1-v3.0.1.tar.gz
-SVT-AV1.FETCH.sha256  = 9a49f13c2f9dd8140e5960d44d9bd66063f7338301e4cce5acbc72c573df6fdd
+SVT-AV1.FETCH.url     = https://github.com/Nj0be/HandBrake-SVT-AV1-PSY/releases/download/contribs/SVT-AV1-PSY-v2.3.0-B.tar.gz
+SVT-AV1.FETCH.sha256  = 104a36fb18c1de8b1221a4b89dd111be473795ea0b2f81484990e00fe7fd9266
 
 SVT-AV1.GCC.args.c_std =
 
